plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/OpenShiftServerEditorModel.java

```diff
@@ -436,7 +437,11 @@ public IStatus redo(IProgressMonitor monitor, IAdaptable adapt) {
 	public void setInitializing(boolean initializing) {
 		this.initializing = initializing;
 	}
-	
+
+	public boolean isInitializing() {
```

Comment:
haven't found any usage of this new method. Can be removed

Reply:
I see it being set from the job that loads the resources:

OpenShiftServerEditorSection#LoadResourcesJob:
```
	private class LoadResourcesJob extends Job {
		...
		@Override
		protected IStatus run(IProgressMonitor monitor) {
			model.setInitializing(true);
			...
			} finally {
				model.setInitializing(false);
			}
			...
```